src/lib/account_update.ts

```diff
@@ -632,7 +632,7 @@ class AccountUpdate implements Types.AccountUpdate {
     undefined;
   account: Precondition.Account;
   network: Precondition.Network;
-  validWhile: Precondition.ValidWhile;
+  globalSlot: Precondition.ValidWhile;
```

Comment:
It's a bit confusing to use `globalSlot` as the public term but keep `ValidWhile` as the internal term in the code. Should this be updated everywhere? Seems like one of those things that doesn't appear to be a big deal but gets annoying and confusing in time.